src/build/tenscript/plan_runner.rs

```diff
@@ -61,7 +64,7 @@ impl PlanRunner {
             GrowCalm =>
                 (GrowStep, 0),
             Shaping =>
-                match self.shape_phase.shaping_step(fabric) {
+                match self.shape_phase.shaping_step(fabric)? {
```

Comment:
Man, the `?` operator is so cool

Reply:
I had to wrestle one out of an iter chain.  Interesting.